docs/pages/reference/meshery-library-errors.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Meshery Lib Errors Reference

```

Comment:
I suggest `Meshery Errors Reference` as this page is about all errors from all Meshery components. As far as I know, there is no `Meshery Library`.

Reply:
```suggestion
title: Meshery Errors Reference

```